tests/conftest.py

```diff
@@ -146,7 +146,22 @@ def img_tbl_exprs(indexed_img_tbl: catalog.Table) -> list[exprs.Expr]:
         t.img.rotate(90).resize([224, 224]),
         t.img.fileurl,
         t.img.localpath,
+    ]
+
+@pytest.fixture(scope='function')
+def sim_exprs_same_idx(indexed_img_tbl: catalog.Table) -> list[exprs.Expr]:
+    t = indexed_img_tbl
+    return [
         t.img.similarity('red truck'),
+        t.img.similarity('red truck', idx='img_idx0')
```

Comment:
why not just leave these in img_tbl_exprs?

Reply:
because i wanted to also verify they have same id. I have reverted this now that i moved the extra checks to another testcase.
<!-- Reviewable comment -OETQXiF77WrwvSK5iIX:bfvpssk -->
<!-- Sent from Reviewable.io -->